Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -461,3 +461,54 @@
         type: Add
         time: 5
         refresh: false
+
+- type: reagent
+  id: BathSalts
+  name: reagent-name-bath-salts
+  group: Narcotics
+  desc: reagent-desc-bath-salts
+  physicalDesc: reagent-physical-desc-powdery
+  flavor: bitter
+  color: "#FAFAFA"
+  metabolisms:
+    Poison:
+      effects:
+      - !type:HealthChange
+        conditions:
+        - !type:ReagentThreshold
+          max: 20
+        damage:
+          types:
+            Asphyxiation: 1.5
+            Poison: 0.5
+    Narcotic:
+      effects:
+      - !type:Jitter
+      - !type:MovespeedModifier
+        walkSpeedModifier: 1.2
+        sprintSpeedModifier: 1.2
+      - !type:Emote
+        emote: drools
+        showInChat: true
+        probability: 0.2
+        conditions:
+        - !type:ReagentThreshold
+          max: 20
+      - !type:GenericStatusEffect
+        key: Stun
+        time: 3
+        type: Remove
+      - !type:GenericStatusEffect
+        key: KnockedDown
+        time: 3
+        type: Remove
+    Medicine:
+      effects:
+      - !type:ResetNarcolepsy
+        conditions:
+        - !type:ReagentThreshold
+          min: 10
+      - !type:ChemVomit
+        probability: 0.2
+        - !type:ReagentThreshold
```

Comment:
There’s something wrong in this particular line.

Reply:
from /Prototypes/Recipes/Reactions/chemicals.yml... i'll fix it